sky/backends/cloud_vm_ray_backend.py

```diff
@@ -846,32 +820,6 @@ def _ibm_handler(blocked_resources: Set['resources_lib.Resources'],
             _add_to_blocked_resources(blocked_resources,
                                       launchable_resources.copy(zone=zone.name))
 
-    # Apr, 2023 by Hysun(hysun.he@oracle.com): Added support for OCI
-    @staticmethod
-    def _oci_handler(blocked_resources: Set['resources_lib.Resources'],
```

Comment:
This is actually still used I think: https://github.com/skypilot-org/skypilot/blob/f5170ee3a1a6711f9ed54941aae2dee4e6525acc/sky/backends/cloud_vm_ray_backend.py#L908-L909

Reply:
OCI has already been moved to new provisioner, so FailoverCloudErrorHandlerV2 should be called instead of this : )

Reply:
Oh, got it. Should we migrate this one as well? Or at least add a TODO for it?

Reply:
Looking at it again, it seems like just using the default handler is probably fine.

Reply:
Yea, the default one should be good enough now, since the new provisioner will handle the output anyway. : )